File: dao_gen/2/csharp/PersonDAO.cs
using System;
using System.Collections.Generic;
using System.Data;
using System.Linq;
using platform.dao;
using platform.dao.param;

namespace com.ctrip.platform.hello
{
    public class PersonDAO : AbstractDAO
    {
        public PersonDAO()
        {
            //注释掉此行或者赋值为string.Empty，然后配置connectionString来直连数据库
            PhysicDbName = "SysDalTest";
            ServicePort = 9000;
            CredentialID = "30303";
            base.Init();
        }

        public IDataReader getHello() {
        {
            try
            {
                IList<IParameter> parameters = new List<IParameter>();
                                
                return this.Fetch("SELECT ID,Address,Name,Telephone,Age,Gender,Birth FROM Person WHERE ", parameters.ToArray());
            }
            catch (Exception ex)
            {
                throw ex;
            }
        }
        
            }
}

File: dal-dao-gen/1/java/PersonDAO.java
package com.ctrip.platform.tools;

import java.sql.ResultSet;
import java.util.ArrayList;
import java.util.List;

import com.ctrip.platform.dao.AbstractDAO;
import com.ctrip.platform.dao.enums.DbType;
import com.ctrip.platform.dao.enums.ParameterDirection;
import com.ctrip.platform.dao.param.StatementParameter;

public class PersonDAO extends AbstractDAO {

	public PersonDAO() {
		logicDbName = "SysDalTest";
		servicePort = 9000;
		credentialId = "30303";
		super.init();
	}

	public ResultSet getAll() {
		List<StatementParameter> parameters = new ArrayList<StatementParameter>();
		return this.fetch("SELECT Birth,Name,Age,Telephone,Gender,Address,ID FROM Person", parameters, null);
	}

	
	
}
